### ISSUE TITLE
Runaway memory use from Builder with large stimuli

### ISSUE THREAD

**Issue by m-macaskill** [psychopy/psychopy#1209]
A user sent me a minimal Builder file that soon after running (just on my computer) lead to an OS error about application memory usage (within a few seconds, the experiment process built up to using 20 GB). The issue appeared to simply be that the text stimuli had a specified size of 72. On my installation, the default settings are for `norm` units. When I set that option to be `pix`, the problem was eliminated.

This occurs on the standalone version 1.83.04 and the current development release.

I can see why the error occurs (a text stimulus 36 times the size of the screen is massive) but am surprised that such a simple and presumably common situation doesn't lead to frequent user issue reports... I seem to recall a long while back that some basic limits checks had been applied to counter this sort of situation arising. If so, have they recently ceased working? If not, do we need to apply some to fix this very fragile error mode?

**Comment by peircej** [comment]
No. Weve talked about the fact that PsychoPy *should* do these checks but it doesn't

This was sent from my phone. Apologies for brevity, typos and inappropriate auto-corrects

-------- Original message --------
